requirements-dev.txt

```diff
@@ -2,5 +2,5 @@
 pytest
 pytest-cov
 coverage
-sphinx=2.2.1
-sphinx_rtd_theme=0.4.3
+sphinx==2.2.1
+sphinx_rtd_theme==0.4.3
```

Comment:
This was leftover from the time when conda didn't accept `==`. Thanks for catching that!